src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Comment.scss';
+
+const Comment = function({ id, name, body, email }) {
+  return (
+    <li key={id} className="Comment">
+      <span>{`Name: ${name} Email: ${email}`}</span>
+      <p>
+        <b>Comment</b>
+        :
+        {' '}
+        {body}
```

Comment:
Тут Миша рассказывал как исправить линтер
```suggestion
        <b>Comment:</b>
        {` ${body}`}
```